cellmlmanip/model.py

```diff
@@ -560,8 +560,30 @@ def get_definition(self, variable):
         return defn
 
     def get_value(self, variable):
-        """Returns the evaluated value of the given variable's RHS."""
-        return float(self.graph.nodes[variable]['equation'].rhs)
+        """Returns the evaluated value of the given variable, as a float."""
```

Comment:
I think we should expand this docstring to emphasise that it recursively evaluates referenced equations and uses initial values for state variables.

We could also merge with the internal helper method, and document the extra parameter as for internal use only.

Reply:
Have updated the docstring, but much happier hiding the argument from the user! It's not part of the public API and also has some hidden constraints: If you were to insert an evaluation for `INa` that would be used, while an evaluation for `V` would not, and if you were to insert evaluations for both states and intermediary values, you'd have to be sure those were consistent or the method behaviour would be undefined